libibverbs/dummy_ops.c

```diff
@@ -396,7 +396,7 @@ static int query_device_ex(struct ibv_context *context,
 	if (attr_size < sizeof(attr->orig_attr))
 		return EOPNOTSUPP;
 
-	memset(&attr->orig_attr, 0, sizeof(attr->orig_attr));
+	memset(attr, 0, sizeof(*attr));
```

Comment:
This is not right, it should be 

memset(attr, 0, attr_size);

Reply:
Why not? We are cleaning all struct without relation to provided size.

Reply:
That will smash the stack, attr_size can be less than sizeof(*attr)

Reply:
OK, thank you.